modules/UI/videolayout/VideoContainer.js

```diff
@@ -495,6 +495,10 @@ export class VideoContainer extends LargeContainer {
 
         stream.attach(this.$video[0]);
 
+        // Ensure large video gets play() called on it when a new stream is attached to it. This is necessary in the
+        // case of Safari as autoplay doesn't kick-in automatically on Safari 15 and newer versions.
+        browser.isWebKitBased() && this.$video[0].play();
```

Comment:
is this also an issue for thumbnails?

Reply:
No, its not an issue for thumbnails, just for large-video.